app/web.ts

```diff
@@ -69,7 +69,7 @@ export default ((publicPath: string) => {
         log.info(`Webserver started on ${port}`);
     });
 
-    const wss: any = new WebSocket.Server({ port: 8080 });
+    const wss: any = new WebSocket.Server({ server, path: '/wss' });
```

Comment:
Given we need to sync this in two files, would it make sense to move it into a constant somewhere?